WinImageBuilder.psm1

```diff
@@ -537,6 +547,22 @@ function Add-VirtIODrivers {
 }
 
 function Add-VirtIODriversFromISO {
+    <#
+    .SYNOPSIS
+     The VirtIO ISO contains all the synthetic drivers for the KVM hypervisor. 
+     This function allows you to add VirtIO drivers to your Windows image, from an ISO file.
+    .DESCRIPTION
+     This function takes the VirtIO drivers from a specified ISO file and installs them into the 
+     given VHD, based on the characteristics given by the image parameter (which contains the
+     image version, image architecture and installation type).

```

Comment:
For adding more context about VirtIO drivers, I would add:

`More info can be found here: https://fedoraproject.org/wiki/Windows_Virtio_Drivers`